addons/pkg/util/clusterbootstrapclone/clusterbootstrapclone.go

```diff
@@ -379,14 +379,33 @@ func (h *Helper) HandleExistingClusterBootstrap(clusterBootstrap *runtanzuv1alph
 			packagesToBeCloned = append(packagesToBeCloned, nonEmptyInlinePackages...)
 			packages = packagesToBeCloned
 		}
+	} else {
+		// packages with values from inline need to be cloned.
```

Comment:
A comment with more details here explaining why this is done would be useful.

e.g. This else is hit in two cases:
1. When no Clusterbootstrap is given by user and clusterbootstraptemplate is cloned as clusterbootstrap but resolved tkr is not set - In this case the providers are already cloned so providers have to be cloned again
2. When full clusterbootstrap without tkr annotation is given by user - In this case we only want to handle inline packages for creating secret. The providers are expected to be present in the clusternamespace i.e. created by user before clusterbootstrap is created.

Reply:
done